src/components/login/Login.tsx

```diff
@@ -0,0 +1,36 @@
+
+import React from 'react'
+
+import './Login.css'
+
+// const Login = ({
+//   handleSubmit,
+//     handleUsernameChange,
+//     handlePasswordChange,
+//     username,
+//     password
+// }) => {
+
+const Login = () => {
+
+  const handleLogin = (event : any) => {
```

Comment:
Can this be more strongly typed? Search on google type for this event.

Reply:
I found this type "const handleLogin = (event: FormEvent<HTMLFormElement>)". I hope that is good.